components/scream/CMakeLists.txt

```diff
@@ -168,6 +168,9 @@ set(SCREAM_TEST_THREAD_INC 1 CACHE STRING "Thread count increment for threaded t
 set (YAMLCPP_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../externals/yaml-cpp)
 set (EKAT_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/ekat)
 
+# Whether to disable warnings from tpls.
+set (SCREAM_DISABLE_TPL_WARNINGS ON CACHE BOOL "")
```

Comment:
I'm puzzled. These lines are already in master. When I run `git diff origin/master CMakeLists.txt` from your branch, I get nothing back. Why is github showing this? Could you please rebase on origin/master? I am thinking that maybe github shows only the diffs against the common anchestor between your branch and master...